types/busboy/index.d.ts

```diff
@@ -1,114 +1,145 @@
-// Type definitions for busboy 0.3
-// Project: https://www.npmjs.com/package/busboy
+// Type definitions for busboy 1.3
+// Project: https://github.com/mscdex/busboy
 // Definitions by: Jacob Baskin <https://github.com/jacobbaskin>
 //                 BendingBender <https://github.com/BendingBender>
+//                 Martin Badin <https://github.com/martin-badin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-import * as http from 'http';
+import { IncomingHttpHeaders } from 'http';
 import { Readable, Writable } from 'stream';
 
-export = busboy;
+declare namespace busboy {
+    interface Limits {
+        /**
+         * Max field name size (in bytes).
+         *
+         * @default 100
+         */
+        fieldNameSize?: number | undefined;
+
+        /**
+         * Max field value size (in bytes).
+         *
+         * @default 1048576 (1MB)
+         */
+        fieldSize?: number | undefined;
 
-declare const busboy: busboy.BusboyConstructor;
+        /**
+         * Max number of non-file fields.
+         *
+         * @default Infinity
+         */
+        fields?: number | undefined;
 
-declare namespace busboy {
-    interface BusboyConfig {
+        /**
+         * For multipart forms, the max file size (in bytes).
+         *
+         * @default Infinity
+         */
+        fileSize?: number | undefined;
+
+        /**
+         * For multipart forms, the max number of file fields.
+         *
+         * @default Infinity
+         */
+        files?: number | undefined;
+
+        /**
+         * For multipart forms, the max number of parts (fields + files).
+         *
+         * @default Infinity
+         */
+        parts?: number | undefined;
+
+        /**
+         * For multipart forms, the max number of header key-value pairs to parse.
+         *
+         * @default 2000 (same as node's http module)
+         */
+        headerPairs?: number | undefined;
+    }
+
+    interface Config {
```

Comment:
can you left alias for backward compatibility (like `type BusboyConfig = Config;`)
the same for all other update/rewritten implementation?

Reply:
yes, sure